Makefile

```diff
@@ -140,6 +144,17 @@ install-server-package:
 
 install: install-server install-ui check-install ## Build and install
 
+install-all: install-server install-ui check-install install-examples ## Build and install, including examples
```

Comment:
Seeing weird behavior here, running `make install` installs the examples packages. Is it supposed to do this? I figure the `install` target should be installing elyra but not the extra packages, hence the two separate targets here for all and not all.

```
(python38) Alans-MacBook-Pro:elyra akchin$ pip list installed | grep elyra
elyra                             3.3.0.dev0
elyra-examples-airflow-catalog    0.0.1
elyra-examples-kfp-catalog        0.0.1
```